extensions/objects/templates/FilepathEditor.js

```diff
@@ -161,6 +161,12 @@ oppia.directive('filepathEditor', [
           $scope.filepaths = response.data.filepaths;
           $scope.filepathsLoaded = true;
         });
+        $scope.width = null;
```

Comment:
It's true, but it feels very not-user-friendly, and tbh I don't think people will know how to work with it. If we go down this route, we probably need to do some user testing.